### DISCUSSION TITLE
rejected github profile achievements

### DISCUSSION THREAD

**Discussion by rezzcode** [Flet/rejected-github-profile-achievements#29] [General]
just imao, i think the `Over 1,000 open issues on public repositories you own.` badge should have been one of the GitHub badges, what do yall think ??

Share your thoughts on this..... ^_^

**Comment by dohyeon5626** [comment]
@the-1Riddle 
I think this is a fantastic idea! Managing over 1,000 open issues is no small feat and it truly highlights the activity and dedication involved in maintaining a project. Such a badge would not only add a bit of fun but also serve as a meaningful recognition of the effort and commitment of contributors. If GitHub were to introduce a badge like this, I believe it would generate a positive response within the community. I would also love to see this badge implemented.

**Reply by rezzcode** [reply]
exactly :100:
